app/controllers/admin_controller.rb

```diff
@@ -102,4 +104,29 @@ def culled_games
   def about
   end
 
+  def stats_display
+    games = Event.current.top_games()
+    titles = Game.where(culled: false).size
+    active_checkouts = Checkout.where(event: Event.current, closed: false).size
```

Comment:
Use `.count`, which is an ActiveQuery method, instead of `.size`. `.size` can cause the query to be executed as it is a method on `Enumerable`, and therefore requires fully-hydrated ActiveRecord objects.